scripts/generate_upgrade_syncset.py

```diff
@@ -0,0 +1,301 @@
+#!/bin/env python
+
+import argparse
+import datetime
+import json
+import logging
+import os
+import re
+import subprocess
+import sys
+import ruamel.yaml as yaml
```

Comment:
I noticed you don't seem to be making any actual use of any of the ruamel-isms in this script. Not that it's really a problem to use it anyway, but I'm curious why you chose this rather than the stock `yaml` lib?

...unless it matters to you that the keys remain in the order you set them? In which case `oyaml` would work too.